test/built-ins/AsyncIteratorPrototype/Symbol.asyncIterator/return-val.js

```diff
@@ -10,11 +10,22 @@ features: [Symbol.asyncIterator, async-iteration]
 ---*/
 
 async function* generator() {}
-var AsyncIteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf(generator.prototype))
+const AsyncIteratorPrototype = Object.getPrototypeOf(Object.getPrototypeOf(generator.prototype))
+const getAsyncIterator = AsyncIteratorPrototype[Symbol.asyncIterator];
 
-var thisValue = {};
+const thisValues = [
+    {},
+    Symbol(),
+    4,
+    4n,
+    true,
+    undefined,
+    null,
+];
 
-assert.sameValue(
-  AsyncIteratorPrototype[Symbol.asyncIterator].call(thisValue),
-  thisValue
-);
+for (const thisValue of thisValues) {
```

Comment:
Is this an ok way of iterating? The repository contains both `for of` and `forEach`:

https://github.com/tc39/test262/blob/6833180066733cce4789d290ddbdfcf6e309226a/test/built-ins/Temporal/TimeZone/basic.js#L25-L29
https://github.com/tc39/test262/blob/6833180066733cce4789d290ddbdfcf6e309226a/test/intl402/PluralRules/prototype/select/non-finite.js#L20-L22

It felt like newer tests leaned more towards `for of`. Is that something to preserve, or eh, whichever way works?

Reply:
As far as I know, we don't have a policy either way, so either is fine. (If we did, I think it should be a linter rule and not dependent on reviewers remembering it, anyway)